go/store/datas/commit.go

```diff
@@ -54,12 +56,13 @@ var valueCommitType = nomdl.MustParseType(`Struct Commit {
 // struct Commit {
 //   meta: M,
 //   parents: Set<Ref<Cycle<Commit>>>,
+//   parentsList: List<Ref<Cycle<Commit>>>,
 //   value: T,
 // }
 // ```
 // where M is a struct type and T is any type.
-func NewCommit(value types.Value, parents types.Set, meta types.Struct) (types.Struct, error) {
-	return commitTemplate.NewStruct(meta.Format(), []types.Value{meta, parents, value})
+func NewCommit(value types.Value, parents types.Set, parentsList types.List, meta types.Struct) (types.Struct, error) {
```

Comment:
Thoughts on this interface, versus taking only the `parentsList` and going `ToSet` here? That seems pretty reasonable...this needs to thread a `context` as well. Current interface means a caller could get it wrong, which isn't great.

Reply:
Is there any benefit in not doing ToSet here?  Is there a time when the elements in the set wouldn't match the elements in the list?  If not seems less error prone to call ToSet here

Reply:
Move it. Much better that way.